participant/app/root.tsx

```diff
@@ -84,6 +87,32 @@ export default function App() {
   // language, this locale will change and i18next will load the correct
   // translation files
   useChangeLanguage(locale);
+  const matomoUrl = MATOMO_URL_BASE || "http://localhost:8080/";
+
+  let isHydrated = useHydrated();
+  if (isHydrated) {
```

Comment:
Let's make this a little more flexible, so that matomo isn't included if the env var isn't set:

```suggestion
  if (isHydrated && MATOMO_URL_BASE) {
```